server/container/guice/cassandra-guice/src/main/java/org/apache/james/CassandraJamesServerMain.java

```diff
@@ -92,20 +92,23 @@
     public static final Module PLUGINS = Modules.combine(
         new CassandraQuotaMailingModule());
 
-    public static final Module CASSANDRA_SERVER_CORE_MODULE = Modules.combine(
+    public static final Module CASSANDRA_SERVER_CORE_MODULE_WITHOUT_CASSANDRA_BLOBSTORE = Modules.combine(
```

Comment:
This look horrible.
I was against the definition of two coarse grain modules because you end up importing a lot of things without even thinking about it.
The other downside is that too big modules do not compose.
The conclusion is that we de-compose (negative composition) and it's so bloated (don't take it for you, @trantienduchn , you just concluded this bad sequence of choices but you are not responsible for the situation).
Can we all agree to go back to finer grain modules that we can compose ?